### ISSUE TITLE
Gods of War

### ISSUE THREAD

**Issue by benbitoun** [jsoma/data-studio-projects#210]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch
President Eisenhower feared that the military industrial complex could lead to a state of perpetual war as the big armament industry will continue to profit from warfare. And the data that the Stockholm International Peace Research Institute (SIPRI) gathered over the years suggest: He was dead on. From the Cold war conflicts to the wars in the middle east, the war on drugs or war on terror: One can tell the history of all the major conflicts since the 1950s by looking at the US weapon supplies list - and that's what I'm doing.

## Summary

I have gathered data about the import and export of weapons from 1950 till 2017. My main source for that was the Stockholm International Peace Research Institute (SIPRI) that kept track of all the exports (type of weaponry, quantity, price etc.):
https://www.sipri.org/databases/armstransfers

Further I found data about all the armed conflicts in the world since 1946 till today - this is a large dataset from the University of Uppsala:
http://www.pcr.uu.se/research/UCDP/

When looking at the data I found that, as a historian, I could play a game and tell immediately which conflict was where between which parties from just looking at the weapon exports from the USA to that country. Now I want to make a nice and compelling visual story out of that.

I also found a nice interactive visualization of that topic in the internet:
![bildschirmfoto 2018-07-24 um 01 06 42](https://user-images.githubusercontent.com/39416122/43118323-70298326-8edf-11e8-855a-c84574be25fe.png)

## Details

**Possible headline(s):** 
The Gods of War: The history of all the worlds conflicts told by looking at US weapon exports
**Data set(s):** 
I have a lot of datasets from SIPRI and from the University of Uppsala
**Code repository:** 
	https://github.com/benbitoun/data-studio/tree/master/Code/03-Gods_of_War_project
**Possible problems/fears/questions:** 
The datasets are large with a lot of columns for the years. A challenge will be how to graph and plot nicely without having to focus on a shorter period of time. I guess finally I can do something other than a bar chart.
## Work so far
I have looked at all the datasets and analyzed the data, made several data frames, cleaned them ans got rid of all the NaN-values.

## Checklist

This checklist must be completed before you submit your draft.

- [x ] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [ x] My issue links to my data set(s)
- [x ] My issue links to my code repository
- [x ] My issue explains what I'd like to explore in the data set
- [x ] My issue includes images - either inspiration or what I've done so far

**Comment by jsoma** [comment]
> The Gods of War: The history of all the worlds conflicts told by looking at US weapon exports

That's a bold title if I've ever seen one!

> When looking at the data I found that, as a historian, I could play a game and tell immediately which conflict was where between which parties from just looking at the weapon exports from the USA to that country. Now I want to make a nice and compelling visual story out of that.

Going along with what Sarah Slobin said in her feedback, I'm excited to see a lot of different graphics for this one. I think a lot of your work is going to be in the storytelling and non-data research end of things this time around - provide us with a compelling image, then give us the history/backstory of what exactly was going on.

**Comment by benbitoun** [comment]
# Update
	
## Your project content: images/words/etc
	
Getting there. I want to do various graphics this time. But I figured out that it might in be better to split the arm exports up by region. So I do South and Central America, Asia, Africa and the Middle East as conflict zones. Starting first with a regular bar graph of the totals of weapon exports. Then do some spaghetti graphs. In Illustrator I want to add meaning to the highest peaks (normally these are wars or military coups).

I have one dataset with the bodycounts in conflicts by year. If anyone could help me how I could add that aspect (scatter?) that would be much appreciated.
![bildschirmfoto 2018-07-27 um 02 42 13](https://user-images.githubusercontent.com/39416122/43305962-935e2ab2-9147-11e8-9a88-c663c7fb3220.png)
Then I am not sure if I should use whole S.America with fine lines (to make it less informative but more artistic) like this:
![bildschirmfoto 2018-07-27 um 00 34 02](https://user-images.githubusercontent.com/39416122/43306038-ceedab34-9147-11e8-8747-9fb1e981a187.png)
Or select the biggest ones:
![bildschirmfoto 2018-07-27 um 00 34 09](https://user-images.githubusercontent.com/39416122/43306057-de59dce6-9147-11e8-8634-8f616d5c9ed0.png)
And same for Central America:
![bildschirmfoto 2018-07-27 um 00 34 23](https://user-images.githubusercontent.com/39416122/43306078-ec64811a-9147-11e8-9656-f1c57315d428.png)

  
## Any changes in direction or topic?
  Not really. Just figured out that I want to break it down and do graphics by region
	
## Problems/Questions
 Coming up with lots of different types of graphics so that the topic want get boring.
	
## Checklist
	
- [x ] I have included my visuals
- [ x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by angelareplica** [comment]
Fascinating and eye-opening topic! I like the fine lines chart for South America because it shows how many countries the U.S. has exported arms to -- it's chaotic but artistic (like you mentioned), and you get a sense of the scale of arms exports. 
However, if you want to extract useful information, the line graphs are too busy: It's hard to follow any of the lines and to tell what's going on. Perhaps you could try making most of the lines grey and highlighting one or two?

For your question about showing body counts in conflict by year, maybe a bar chart with a line graph plotted on top? That way, you can see if the height of the bars correlates with the line(s). 

Awesome work - really excited to see how this turns out.

**Comment by cfelke** [comment]
I agree with you that the line graph looks more artsy and aesthetic (which I like) but it may be too confusing in the end. I'd also opt for highlighting the countries which are the most important ones in your analysis (like Angela suggested) and make some annotations explaining why we see these peaks. 

Following this overview graph you could zoom in on one or two countries and discuss the arms transfers in detail, either in various graphs or in one (The Pudding, NYT often do this). I'd suggest playing with zooming in and out to show different aspects of your data, in general.
Regarding the body counts, just generally speaking, I for one would opt for a considerate display of deaths (e.g. no plain bar graph) because we're talking about people who died in a conflict/war after all.

This is an important topic, looking forward to see your update.

**Comment by castorsia** [comment]
Important, interesting and always pertinent topic. I agree on your choice to break it down by region. The graphs indeed look very elegant-but as you said, they are difficult to convey the story. Keep it, because I think breaking it down even more can lead you out of your conundrum. After one spaghetti graph (haha, I liked that), you can graph one country... Say, Iran, after the ME. That way you can provide context and have the best of two worlds. Great work!

**Comment by sarahslo** [comment]
i'm thinking heat map for this one. love the thesis by the way.

issue with the fever lines, besides there are too many to read, is that the data points don't really connect, how many arms were exported one year to the next don't need to connect. and is this the count of arms? not clear what the numbers stand for. is this counting shipments? it's not like a shipment of bullets is the same as a shipment of bombs. what are you counting?

anyway, heat map may get us there faster. give it a shot.

**Comment by benbitoun** [comment]
# Update
I was working on some visuals and thinking how I could keep the nice looking chaos of massive weapon transfers and making it more readable at the same time. Then I saw Sarahs feedback about heat map and now want to figure out what a heat map actually is...

Because of the limited time issue and also to make it more accessible for readers I gave it a focus and decided not to go for once with the middle east but tell the story of U.S interventionism in South America by looking at the arm transfers.
Anyway, here's what I have done with this. 

Please  note that I will add the Source later in the html file. And I will do everything left (titles) and explain what the numbers stand for prominently in the subhead.

	
## Your project content: images/words/etc
	
![bildschirmfoto 2018-08-01 um 00 39 55](https://user-images.githubusercontent.com/39416122/43510501-da90fb9e-9543-11e8-878a-b60ee16ba28f.png)
 
This just for fun:
![bildschirmfoto 2018-08-01 um 00 45 44](https://user-images.githubusercontent.com/39416122/43510546-f7dbd2b4-9543-11e8-82b9-57846c6df65b.png)
![bildschirmfoto 2018-08-01 um 04 20 38](https://user-images.githubusercontent.com/39416122/43510556-fd35ee3e-9543-11e8-84ee-67c7cd5472aa.png)
 breaking it down:
![bildschirmfoto 2018-08-01 um 04 33 39](https://user-images.githubusercontent.com/39416122/43510574-15c79330-9544-11e8-8914-e8929e03c8d0.png)

## Any changes in direction or topic?
  Mentioned above
	
## Problems/Questions
 Illustrator without knowing what I'm doing and without a mouse
	
## Checklist
	
- [ x] I have included my visuals
- [x ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by julialedur** [comment]
I love your idea, the topic is really interesting! 

I really like the style and the color scheme of your graphs, but I'm not sure if area charts are the best form of visualization. Even though they're a clear way to show the peaks and annotate, I feel like in this case they are a little too busy because there are too many countries plotted together. I'd go for a heatmap or a sankey chart (I'm attaching a picture below). 

![graph](https://user-images.githubusercontent.com/39416238/44005831-be893fda-9e47-11e8-9709-9b11841c69ee.JPG)

There's a library you can use to make them. Here's a good tutorial I found: https://github.com/anazalea/pySankey 

Other small details: 
- in your second graph, I would make all annotations white. Colored text on a black background is a pain to read. 
- in your third graph, I'm not sure what the axes are representing. You might want to add labels. 

Looking forward to seeing your next update! :)

**Comment by jessimckenzi** [comment]
OOOOhh I love Julia's idea about the Sankey graphs! A series of those representing different decades (or other chunk of time) would be a good way to show how weapons flow changed over time as conflicts emerge/dissipate. Really really cool topic. My favorite graph is the second, and it looks really informative (although the text is too small for me to read so I don't actually know).
